src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/template.json

```diff
@@ -46,7 +46,7 @@
     {
       "id": "openInEditor",
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "description": "Opens the created controller in the editor",
+      "description": "Opens ValueController.cs in the editor",
```

Comment:
1. This name should be automatically updated to reflect the name of the file a user chooses.
2. Since this is an item template, the description should be about that item.

Reply:
@phenning do you mean my point (1) is incorrect❔ If so, ASP.NET has a larger problem because most of our item templates use the specific filename from `primaryOutputs`.

Reply:
Yea, it will work in a primary output. I'm talking specifically about the description field here, as I don't think the descriptions here get run though the generators, I think they are meant to be static strings.

Reply:
Thinking about this some more I don't believe that the post action descriptions here ever actually get shown to a user by any of our existing hosts. I think this is more for informational purposes to the template author.  @vlada-shubina can confirm

Reply:
But to your point #2, I agree it should be about that item, but the item name is changeable at the time of creation and may not be called ValueController.cs when the user actually creates the file, so the original "created controller" is accurate from a different point of view.

In any event, this string will not be surfaced to the user.

Reply:
> In any event, this string will not be surfaced to the user.

Ah, you heard back from @vlada-shubina 😄 In any case, I just tried to create a couple of items from our older templates that use the `primaryOutputs` name in the post-action `description` and confirmed this.

To be doubly sure, there isn't any logging somewhere that uses this resource (or a translation) in its output❔ I checked the output windows I have available in VS 17.6 Int Preview but, wasn't sure if there are options to turn something else on.

Reply:
Reverted these changes done in next iteration&hellip;

Reply:
description may be shown to use in CLI context, if the post action fails. 
Therefore it is normally localised.

However, this post action doesn't run in CLI, so @phenning is correct here, but mind that it only applies when post action is not used in CLI.